From b28e6343f3f0aa89b336f4249904429c57035026 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 15 Sep 2018 20:52:44 +0200
Subject: [PATCH 1/1] efi_selftest: missing braces in
 efi_selftest_textinputex.c

gcc 4.8.4 wants to see all levels of braces when initializing a
structure to zeros.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_selftest/efi_selftest_textinputex.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/efi_selftest/efi_selftest_textinputex.c b/lib/efi_selftest/efi_selftest_textinputex.c
index d20d8ad89d..de44224ce1 100644
--- a/lib/efi_selftest/efi_selftest_textinputex.c
+++ b/lib/efi_selftest/efi_selftest_textinputex.c
@@ -116,7 +116,7 @@ static int teardown(void)
  */
 static int execute(void)
 {
-	struct efi_key_data input_key = {0,};
+	struct efi_key_data input_key = { {0, 0}, {0, 0} };
 	efi_status_t ret;
 	efi_uintn_t index;
 
-- 
2.18.0

